_posts/en/posts/2016-08-23-release-0-13-0.md

```diff
@@ -215,7 +215,7 @@ These are the SHA-256 hashes of the released files:
 [report any problems]: https://github.com/bitcoin/bitcoin/issues/new
 [release notes]: /en/releases/0.13.0/
 [release notes arm]: /en/releases/0.13.0/#linux-arm-builds
-[release notes changelog]: /en/releases/0.13.0/#change-log
+[release notes changelog]: /en/releases/0.13.0/#notable-changes
```

Comment:
The existing link also works here on the current site.  I think you want to use the same trick as mentioned elsewhere, changing the source file to use `## 0.13.0 Change log {#change-log}` to make the anchor static.

Reply:
Fixed, and updated #686 so we'll see if Travis is happy there too.